include/inexor/vulkan-renderer/imgui/imgui.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
```

Comment:
General note:
Many parameter uses `const`, but we decided not to use `const` in header.

Reply:
Right. I will fix this for `imgui.hpp` and add a ticket for the other files.

Reply:
I will review the entire code in another PR: https://github.com/inexorgame/vulkan-renderer/issues/239.